lib/hammer_cli_foreman/virtual_machine.rb

```diff
@@ -0,0 +1,57 @@
+
+module HammerCLIForeman
+
+  class VirtualMachine < HammerCLIForeman::Command
+    resource :compute_resources
+    command_name 'virtual_machine'
+    desc _("View and manage compute resource's virtual machines")
+
+    module ComputeResourceOptions
+
+      def self.included(base)
+        base.build_options
+
+        base.validate_options do
+          any(:option_compute_resource_id, :option_compute_resource_name).required
+        end
+      end
+
+    end
```

Comment:
This is not used anywhere. Is that a leftover?

Reply:
You are very right, I added it but did not use it eventually. I will delete this part.